lib/datadog/tracing/contrib/ethon/easy_patch.rb

```diff
@@ -58,6 +59,12 @@ def complete
                     set_span_error_message("Request has failed with HTTP error: #{response_code}")
                   end
                 end
+
+                @datadog_span.set_tags(
+                  Datadog.configuration.tracing.header_tags.response_tags(
+                    Core::Utils::Hash::CaseInsensitiveWrapper.new(parse_response_headers)
+                  )
+                )
```

Comment:
If `header_tags` is not configured, this would be a no-op, right?

Reply:
Same for the `request_tags`

Reply:
That is correct, both methods return an empty list, and `#set_tags` naturally does nothing with the empty list.